nym-vpn-core/crates/nym-vpn-proto/proto/nym_vpn_service.proto

```diff
@@ -241,19 +259,27 @@ message Probe {
 
 message GatewayResponse {
   GatewayId id = 1;
-  Location location = 2;
+  RichLocation location = 2;
   Probe last_probe = 3;
   optional Score wg_score = 4;
   optional Score mixnet_score = 5;
   string moniker = 6;
   repeated string exit_ipv4s = 7;
   repeated string exit_ipv6s = 8;
   optional string build_version = 9;
+  optional Performance wg_performance_v2 = 10;
```

Comment:
Why the `_v2` prefix since there is no existing `wg_performance` field in the current proto?

Reply:
Not sure exposing this prefix is relevant for grpc and all the different layers build ontop of it (types, uniffi, rpcClient etc)